webui/react/src/components/LogViewer/LogViewer.tsx

```diff
@@ -176,7 +178,7 @@ const LogViewer: React.FC<Props> = ({
 
     const buffer: Log[] = [];
 
-    local.current.isFetching = true;
+    setIsFectching(true);
 
     await readStream(
       onFetch({ limit: PAGE_LIMIT, ...config } as FetchConfig, type),
```

Comment:
I think that, maybe, we have to re-write some parts of this component... the `onFetch` is `?` and, the only place that it actually isn't being passed (as far as I could find) is the `TrialDetailsLogs` component, because the `trial` is undefined while initial fetching... also, as it is, it's kind of confusing imho.